src/tests/FlapperUniV2.t.sol

```diff
@@ -253,7 +274,7 @@ contract FlapperUniV2Test is DssTest {
     }
 
     function testAuthModifiers() public virtual {
-        WardsLike(address(flapper)).deny(address(this));
+        vm.prank(PAUSE_PROXY); WardsLike(address(flapper)).deny(address(this));
```

Comment:
I don't think we need this anymore, as now the test is not relied.

Reply:
Replaced with assert